jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java

```diff
@@ -0,0 +1,221 @@
+		TRANSIENT_FAIL_PINNED_NATIVE(Pinned.NATIVE),
+		/** Transient fail due to continuation pinned {@link Pinned#MONITOR} */
+		TRANSIENT_FAIL_PINNED_MONITOR(Pinned.MONITOR);
+
+		final Pinned pinned;
+
+		/**
+		 * @return 
+		 **/
+		public Pinned pinned() {
+			return pinned;
+		}
+
+		private PreemptStatus(Pinned reason) {
+			this.pinned = reason;
+		}
+	}
+
+
+	/**
+	 * Public constructor for Continuation
+	 * @param scope The scope of the Continuation
+	 * @param target The target of the Continuation to execute
+	 */
+	public Continuation(ContinuationScope scope, Runnable target) {
+		throw new UnsupportedOperationException();
+	}
+
+	/**
+	 * @return
+	 */
+	public ContinuationScope getScope() {
+		throw new UnsupportedOperationException();
+	}
+	
+	/**
+	 * @return
+	 */
+	public Continuation getParent() {
+		throw new UnsupportedOperationException();
+	}
+	
+	/**
+	 * @param scope
+	 * @return
+	 */
+	public static Continuation getCurrentContinuation(ContinuationScope scope) {
+		throw new UnsupportedOperationException();
+	}
+	
+	/**
+	 * @return
+	 */
+	public StackWalker stackWalker() {
+		throw new UnsupportedOperationException();
+	}
+
+	/**
+	 * @param options
+	 * @return
+	 */
+	public StackWalker stackWalker(Set<StackWalker.Option> options) {
+		throw new UnsupportedOperationException();
+	}
+
+	/**
+	 * @param options
+	 * @param scope
+	 * @return
+	 */
+	public StackWalker stackWalker(Set<StackWalker.Option> options, ContinuationScope scope) {
+		throw new UnsupportedOperationException();
+	}
+
+	/**
+	 * @return
+	 */
+	public StackTraceElement[] getStackTrace() {
+		throw new UnsupportedOperationException();
+	}
+
+	/**
+	 * @param <R>
+	 * @param cont
+	 * @param scope
+	 * @param sup
+	 * @return
+	 */
+	public static <R> R wrapWalk(Continuation cont, ContinuationScope scope, Supplier<R> sup) {
+		throw new UnsupportedOperationException();
+	}
+
+	/**
+	 * 
+	 */
+	public final void run() {
+		throw new UnsupportedOperationException();
+	}
+
+	/**
+	 * @param scope
+	 * @return
+	 */
+	public static boolean yield(ContinuationScope scope) {
+		throw new UnsupportedOperationException();
+	}
+
+	/**
+	 * @param reason
+	 */
+	protected void onPinned(Pinned reason) {
+		throw new UnsupportedOperationException();
+	}
+
+	/**
+	 * 
+	 */
+	protected void onContinue() {
+		throw new UnsupportedOperationException();
+	}
+
+	/**
+	 * @return
+	 */
+	public boolean isDone() {
+		throw new UnsupportedOperationException();
+	}
+
+	/**
+	 * @return
+	 */
+	public boolean isPreempted() {
+		throw new UnsupportedOperationException();
+	}
+
+	/**
+	 * 
+	 */
+	public static native void pin();
+
+	/**
+	 * 
+	 */
+	public static native void unpin();
+
+	/**
+	 * @param scope
+	 * @return
+	 */
+	public static boolean isPinned(ContinuationScope scope) {
+		throw new UnsupportedOperationException();
+	}
+
+	/**
+	 * @param t
+	 * @return
+	 */
+	public PreemptStatus tryPreempt(Thread t) {
+		throw new UnsupportedOperationException();
+	}
+}
```

Comment:
> 22:27:05  jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java:47: trailing whitespace.
22:27:05  +	 * 
22:27:05  jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java:68: trailing whitespace.
22:27:05  +		 * @return 
22:27:05  jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java:95: trailing whitespace.
22:27:05  +	
22:27:05  jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java:102: trailing whitespace.
22:27:05  +	
22:27:05  jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java:110: trailing whitespace.
22:27:05  +	
22:27:05  jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java:154: trailing whitespace.
22:27:05  +	 * 
22:27:05  jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java:176: trailing whitespace.
22:27:05  +	 * 
22:27:05  jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java:197: trailing whitespace.
22:27:05  +	 * 
22:27:05  jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java:202: trailing whitespace.
22:27:05  +	 * 

These trailing whitespace need to be removed.